components/motor/fake/motor.go

```diff
@@ -351,8 +343,9 @@ func (m *Motor) GoTo(ctx context.Context, rpm, pos float64, extra map[string]int
 	if err != nil {
 		return err
 	}
-	if curPos == pos {
-		return nil
```

Comment:
Is this changing behaviour? If it was zero previously, it would just return from GoTo, and now it errors. The response change changes the behaviour when this is used in the client code or another driver. 

THis is a fake motor so not super important, but always keep an eye out for this sort of change, it is breaking functionality, if not our own API.

Reply:
Ah no, now I see the err != nil check.